### PR TITLE
Warning to properly set camera folder when enabling camera uploads

### PR DESCRIPTION
**Pull request by davigonz**
Implements #2303

### CODE REVIEW

File: src/com/owncloud/android/ui/activity/Preferences.java [hunk 1]
```diff
@@ -543,27 +537,24 @@ private void toggleCameraUploadsVideoOptions(Boolean initializing, Boolean isChe
         if (isChecked) {
             mPrefCameraUploadsCategory.addPreference(mPrefCameraVideoUploadsWiFi);
             mPrefCameraUploadsCategory.addPreference(mPrefCameraVideoUploadsPath);
+            showSimpleDialog(getString(R.string.proper_videos_folder_warning_camera_upload),
```
[the remark this answers is not in the document]
Comment (davigonz) [reply src/com/owncloud/android/ui/activity/Preferences.java]: Yes, done

File: src/com/owncloud/android/ui/activity/Preferences.java [hunk 2]
```diff
@@ -492,38 +491,33 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
...
      *                     view and showing it when the user just unchecked the feature checkbox
      * @param isChecked    camera uploads for pictures is checked
      */
     private void toggleCameraUploadsPictureOptions(Boolean initializing, Boolean isChecked) {
         if (isChecked) {
             mPrefCameraUploadsCategory.addPreference(mPrefCameraPictureUploadsWiFi);
             mPrefCameraUploadsCategory.addPreference(mPrefCameraPictureUploadsPath);
-
-
+            showSimpleDialog(getString(R.string.proper_pics_folder_warning_camera_upload),
```
[the remark this answers is not in the document]
Comment (davigonz) [reply src/com/owncloud/android/ui/activity/Preferences.java]: Done

**Comment by jesmrec** [comment]
Let's move to QA

**Comment by jesmrec** [comment]
Approved

**Pull request merged**